File: sdks/thunkmetrc-go/wrapper/models/vehicle.go
package models

type Vehicle struct {
    FacilityId int64 `json:"FacilityId,omitempty"`
    Id int64 `json:"Id,omitempty"`
    IsArchived bool `json:"IsArchived,omitempty"`
    LastModified string `json:"LastModified,omitempty"`
    LicensePlateNumber string `json:"LicensePlateNumber,omitempty"`
    Make string `json:"Make,omitempty"`
    Model string `json:"Model,omitempty"`
}

File: sdks/thunkmetrc-rs/wrapper/src/models/vehicle.rs
use serde::{Deserialize, Serialize};
#[allow(unused_imports)]
use super::*;

#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct Vehicle {
    #[serde(rename = "FacilityId", skip_serializing_if = "Option::is_none")]
    pub facility_id: Option<i64>,
    #[serde(rename = "Id", skip_serializing_if = "Option::is_none")]
    pub id: Option<i64>,
    #[serde(rename = "IsArchived", skip_serializing_if = "Option::is_none")]
    pub is_archived: Option<bool>,
    #[serde(rename = "LastModified", skip_serializing_if = "Option::is_none")]
    pub last_modified: Option<String>,
    #[serde(rename = "LicensePlateNumber", skip_serializing_if = "Option::is_none")]
    pub license_plate_number: Option<String>,
    #[serde(rename = "Make", skip_serializing_if = "Option::is_none")]
    pub make: Option<String>,
    #[serde(rename = "Model", skip_serializing_if = "Option::is_none")]
    pub model: Option<String>,
}
